# Author: Ehsan Moravveji (VSCentrum, KU Leuven)
# Purpose: The 'cstdint' library needs to be included for exposing the 'uint64_t' type

diff -ruN kineto-orig/libkineto/src/SampleListener.h kineto/libkineto/src/SampleListener.h
--- kineto-orig/libkineto/src/SampleListener.h	2026-01-09 13:13:20.293236000 +0100
+++ kineto/libkineto/src/SampleListener.h	2026-01-09 13:17:10.075117000 +0100
@@ -7,6 +7,7 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 namespace KINETO_NAMESPACE {
 
